tests/manager/models/conftest.py

```diff
@@ -0,0 +1,47 @@
+# conftest.py
+from uuid import UUID
+
+import pytest
+from graphene import Schema
+from graphene.test import Client
+
+from ai.backend.common.utils import update_nested_dict
+from ai.backend.manager.models.gql import GraphQueryContext, Mutations, Queries
+
+
+@pytest.fixture(scope="module", autouse=True)
+def client() -> Client:
+    return Client(Schema(query=Queries, mutation=Mutations, auto_camelcase=False))
+
+
+@pytest.fixture(scope="function")
+def get_base_context():  # noqa: F811
```

Comment:
"Is the name get_base_context() suitable for this function, or would you suggest a different name?"